docs/learning/api-support-policies.md

```diff
@@ -84,8 +84,8 @@ Very rarely, a critical security issue may arise that cannot reasonably be addre
 | ------------- | --------------- | --------------- | --------------- | ----------------- | --------------- |
 | 1.x           | **End of life** | 2019-06-03      | 2020-05-07      | n/a               | 2020-11-01      |
 | 2.x           | **Maintenance** | 2020-05-07      | 2022-01-24      | 2022-12-31        | 2023-3-31       |
-| 3.x           | **Current**     | 2022-01-24      | **_2023-3-31_** | **_2023-9-30_**   | **_2024-3-31_** |
-| 4.x           | **Future**      | **_2023-3-31_** | -               | -                 | -               |
+| 3.x           | **Current**     | 2022-01-24      | **_2023-5-22_** | **_2023-11-22_**  | **_2024-5-22_** |
```

Comment:
3.x = Active, 4.x = Current ?

Reply:
You changed 3.x to Maintenance. Should be Active. The definitions are above this table.

Reply:
Sorry copy and paste, its been fixed